broadway/api/utils/bootstrap.py

```diff
@@ -91,6 +91,7 @@ def initialize_course_tokens(settings: Dict[str, Any], flags: Dict[str, Any]):
     for course_id, tokens in courses.items():
         course = Course(id_=course_id, tokens=tokens)
         course_dao.insert_or_update(course)
+        settings["QUEUE"].add_queue(course_id)
```

Comment:
Why are we doing this in two places? Here and in `tests/api/_utils/database.py:initialize_db()`?

Reply:
Nvm... That is a test lol